client/components/section-nav/index.jsx

```diff
@@ -63,6 +64,7 @@ class SectionNav extends Component {
 		return (
 			<div className="section-nav__mobile-header" onClick={ this.toggleMobileOpenState }>
 				<span className="section-nav__mobile-header-text">{ this.props.selectedText }</span>
+				<Gridicon icon="chevron-down" size={ 18 } />
```

Comment:
This line is breaking SectionNav tests. I think we'll need to update this line https://github.com/Automattic/wp-calypso/blob/master/client/components/section-nav/test/index.jsx#L53 to:

```javascript
headerTextElem = headerElem.props.children[ 0 ];
```

Reply:
Done!